app/services/katello/candlepin/message_handler.rb

```diff
@@ -85,6 +85,13 @@ def import_pool(index_hosts = true)
           ::Katello::Pool.import_pool(pool_id, index_hosts)
         end
       end
+
+      def delete_pool
+        if pool
+          Rails.logger.warn "deleting pool #{pool.id} from Katello"
```

Comment:
This is probably OK at info level instead of warn. There's no action for the user to take if they see the message.